assets/js/wpjm-stats.js

```diff
@@ -0,0 +1,78 @@
+/* global job_manager_stats */
+( function () {
+	// From https://youmightnotneedjquery.com/#ready
+	function ready( fn ) {
+		if ( document.readyState !== 'loading' ) {
```

Comment:
WordPress has a built-in package for this: [@wordpress/dom-ready](https://github.com/WordPress/gutenberg/blob/trunk/packages/dom-ready/README.md#)

Reply:
Is this available in the frontend or should it be a build dependency?

Reply:
Yes no extra steps needed, webpack will generate a `wpjm-stats.asset.php` for the script with the list of WP dependencies, and we add them to the wp_register_script call in [WP_Job_Manager::register_script](https://github.com/Automattic/WP-Job-Manager/blob/2be6144c8014f988873f9bdea43a3569a7048f9e/includes/class-wp-job-manager.php#L322)